cartography/intel/semgrep/findings.py

```diff
@@ -0,0 +1,217 @@
+import logging
+from typing import Any
+from typing import Dict
+from typing import List
+
+import neo4j
+import requests
+
+from cartography.client.core.tx import load
+from cartography.graph.job import GraphJob
+from cartography.models.semgrep.findings import SemgrepDeploymentSchema
+from cartography.models.semgrep.findings import SemgrepSCAFindingSchema
+from cartography.models.semgrep.findings import SemgrepSCALocationSchema
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+
+@timeit
+def get_deployment(semgrep_app_token: str) -> List[Dict[str, Any]]:
```

Comment:
Since the deployment will only be a single object, I think we should refactor to refactor to not return a list.